cmd/state/errors.go

```diff
@@ -101,13 +101,6 @@ func unwrapError(err error) (int, error) {
 		return code, nil
 	}
 
-	// Log error if this isn't a user input error
-	if !locale.IsInputError(err) {
-		multilog.Critical("Returning error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
-	} else {
-		logging.Debug("Returning input error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
-	}
```

Comment:
These are actual errors being logged. It should not be removed.

Reply:
Got it. I was honestly unsure about those lines. I thought perhaps they would be ultimately logged. Thanks for pointing out my error though.